### PR TITLE
add app.rb, main.rb file implement ui

### PR DESCRIPTION
**Pull request by YannickZahinda**
### OOP_school_library add basic UI

In this project I performed the complete UI for the School library

### CODE REVIEW

File: app.rb [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+    puts 'Enter person ID:'
+    person_id = gets.chomp.to_i
+    puts "List of rentals for person id #{person_id}:"
+    @rentals.each { |rental| puts rental.to_s if rental.person.id == person_id }
+  end
+
+  def run
+    loop do
+      option = show_menu
+      case option
```
Comment (Davidosky007) [inline app.rb:105]: * Kindly fix this error i get when i try to create a teacher specialization 
image

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
Hi @YannickZahinda 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
* Kindly update your readme with the right command to start your App, as you have this command `ruby app.rb` instead of `ruby main.rb` 
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app.rb (hunk 1 above)
Comment (YannickZahinda) [reply app.rb:105]: Hello @Davidosky007, 
Great review performance.

**Review by EricMbouwe — APPROVED** [review]
Hi @YannickZahinda, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- Code is running fine
- No linters errors
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**